node/test_network.go

```diff
@@ -81,6 +81,7 @@ func NewTestNetwork(tb testing.TB, conf config.Config, l log.Log, size int) []*T
 		// Wait for nodes to shutdown
 		g.Wait()
 
+		// nolint:usetesting
```

Comment:
Why not using `tb.Context()` here?

Reply:
`tb.Context()` is canceled before `tb.Cleanup` so if we would use that we would pass an already canceled context to `a.Cleanup`.